TypeCobol.LanguageServer/Completion Factory/CompletionFactory.cs

```diff
@@ -587,11 +587,14 @@ public static IEnumerable<CompletionItem> GetCompletionForTo(FileCompiler fileCo
                     }
                 }
 
-                if (sendingVar?.RepeatedCharacterValue?.Token != null &&
-                    (sendingVar.RepeatedCharacterValue.Token.TokenType == TokenType.SPACE
-                     || sendingVar.RepeatedCharacterValue.Token.TokenType == TokenType.SPACES))
+                var tokenType = sendingVar?.RepeatedCharacterValue?.Token?.TokenType;
+                if (tokenType != null)
```

Comment:
At first I did not understand how this test would work since `TokenType` is a ValueType (enum) but in fact here the variable `tokenType` is of type `Nullable<TokenType>` so this test is equivalent to `if (tokenType.HasValue)`. I think this is clearer than comparing to null but it is ok.